governance/votes/Executive Vote - November 12, 2021.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Template - [Executive Vote] Parameter Changes, Return Unused Budget to the Surplus Buffer  - November 12, 2021
+summary: GUNIV3DAIUSDC-A, WSTETH-A, and DIRECT-AAVEV2-DAI parameter changes and return unused funds from GovAlpha to the Surplus Buffer
+date: 2021-11-12T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Parameter Changes, Return Unused Budget Funds to the Surplus Buffer  - November 12, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Changes to the GUNIV3DAIUSDC-A [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam), [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio), and [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee), as detailed below
+- [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) parameter for WSTETH-A will be increased to 50 million DAI
+- Changes to the DIRECT-AAVEV2-DAI [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam), [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam), and Target Borrow Rate, as detailed below
+- 218,059.10 DAI will be returned to the Surplus Buffer from PauseProxy
```

Comment:
I would add a little more detail here? ... unused DAI from the GovAlpha Core Unit budget will be returned...

Would also say 'PauseProxy contract'